models/README.md

```diff
@@ -0,0 +1,8 @@
+This directory holds all model files which are required by facerecognition app to work properly.
+
```

Comment:
Hi @stalker314314 
A link here as a reference may be enough,,

Ideally in models/1 along with the files, but it can be generic. Maybe something like that here:

 * The models used for facecerecognition are extracted from https://github.com/davisking/dlib-models
 * General license: https://github.com/davisking/dlib-models/blob/master/LICENSE
 * See details on: https://github.com/davisking/dlib-models/blob/master/README.md

Reply:
Yes, reference outside of main README.md seems better. However, I opted to create separate, LICENCE file in `models/` dir, not use eixsting `models/README.md`. If you agree, merge pull request please:) If not, I can move to `models/README.md`.